src/esp/geo/VoxelWrapper.h

```diff
@@ -3,11 +3,18 @@
 
 #include "VoxelGrid.h"
 #include "esp/assets/ResourceManager.h"
+#include "esp/gfx/DrawableGroup.h"
 #include "esp/scene/SceneNode.h"
 
 namespace esp {
 namespace geo {
 
+// When an object is voxelized, it is given a VoxelWrapper which points to an
```

Comment:
Will a class comment like this get picked up correctly by the doc-generator?

E.g. https://aihabitat.org/docs/habitat-sim/classesp_1_1assets_1_1ResourceManager.html